lib/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var watcherServerMiddleware = require('./middleware');
+var serveAssetMiddleware = require('./serve-asset-middleware');
+var watcherMiddleware = require('./watcher-middleware');
+
+module.exports = {
+  'watcherServerMiddleware': watcherServerMiddleware,
```

Comment:
It was more for maintaining backward compatibility. If we need to do a point release of ember-cli that is using the combined middleware and requires a bug fix in broccoli-middleware.